core/src/main/scala/filodb.core/memstore/MemStore.scala

```diff
@@ -139,6 +140,12 @@ trait MemStore extends ChunkSource {
    */
   def indexValues(dataset: DatasetRef, shard: Int, indexName: String, topK: Int = 100): Seq[TermInfo]
 
+  def indexValuesWithFilters(dataset: DatasetRef, shard: Int, filters: Seq[ColumnFilter],
+                          column: String, end: Long, start: Long, limit: Int): List[ZeroCopyUTF8String]
+
+  def partKeysWithFilters(dataset: DatasetRef, shard: Int, filters: Seq[ColumnFilter],
```

Comment:
Need ScalaDoc here.  As we discussed this should be returning TSPartition

Reply:
Redesigned the method so that it returns TimeSeriesPartiton iterator, added ScalaDoc with summary

Reply:
I don't see the scaladoc?